scripts/experiment_1/nbconverted/1_train_vae.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Train VAE
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import ast
+import pandas as pd
+import numpy as np
+import random
+import warnings
+warnings.filterwarnings(action='ignore')
+
+from functions import vae
+
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+# In[2]:
+
+
+# Create list of base directories
+
+base_dir = os.path.abspath(os.path.join(os.getcwd(),"../.."))
+
+base_dirs = [os.path.join(base_dir, "data"),
+             os.path.join(base_dir, "models"),
+             os.path.join(base_dir, "output"),
+             os.path.join(base_dir, "data", "encoded"),
+             os.path.join(base_dir, "output", "stats"),
+             os.path.join(base_dir, "output", "viz")
+             ]
+
+# Check if analysis directory exist otherwise create
+
+for each_dir in base_dirs:
+
+    if os.path.exists(each_dir):
+        print('directory already exists: {}'.format(each_dir))
+    else:
+        print('creating new directory: {}'.format(each_dir))
+    os.makedirs(each_dir, exist_ok=True)
+
+
+# In[10]:
+
+
+# Load config params
+config_file = "config_exp_1.txt"
+
+d = {}
+float_params = ["learning_rate", "kappa", "epsilon_std"]
+str_params = ["analysis_name", "NN_architecture"]
+lst_params = ["num_batches"]
+with open(config_file) as f:
+    for line in f:
+        (name, val) = line.split()
+        if name in float_params:
+            d[name] = float(val)
+        elif name in str_params:
+            d[name] = str(val)
+        elif name in lst_params:
+            d[name] = ast.literal_eval(val)
+        else:
+            d[name] = int(val)
+
+
+# In[4]:
+
+
+# Parameters
+num_dims = d["num_dims"]
+
+
+# In[5]:
+
+
+# Load arguments
+normalized_data_file = os.path.join(
+    os.path.abspath(os.path.join(os.getcwd(),"../..")),
+    "data",
+    "input",
+    "train_set_normalized.pcl")
+
+
+# In[6]:
+
+
+# Read data
+normalized_data = pd.read_table(
```

Comment:
I am noticing that they are always transposed. Is there any reason why they are not saved in a transposed form?

Reply:
This is the dimensions of the downloaded data.  So each time it is read it it gets transposed.  I understand that I would be doing this each time I read in the data, I think the rationale here is that I wanted to make sure people knew that this was the raw data that was downloaded and that the only change was that it was transposed